acme/acme/messages.py

```diff
@@ -27,6 +27,8 @@ class Error(jose.JSONObjectWithFields, errors.Error):
             ('dnssec', 'The server could not validate a DNSSEC signed domain'),
             ('invalidEmail',
              'The provided email for a registration was invalid'),
+            ('invalidContact',
+             'The provided email for a registration was invalid'),
```

Comment:
nit: perhaps the description here should be more general to match the error type. Kuba wanted to keep these error messages in sync with the ACME, but that was a bit contentious (see #1785). With this in mind I'd like to see this being something like either:
```
The contact URI for a registration was invalid
```
or
```
The provided contact information for a registration was invalid
```
I have a slight preference for the former as just including the string from the ACME spec is in line with most of error messages here, but if you prefer the latter, that's fine with me.